### ISSUE TITLE
Overview doesn't support code folding

### ISSUE THREAD

**Issue by Yanpas** [geany/geany-plugins#268]
If I fold some peace of code in editor it will be still visible in overview plugin

Geany and Overview are latest, from git (built 1 week ago)

**Comment by Johndeep** [comment]
I agree on that, which is especially annoying, because the overview scrolling stops working correctly, as the line number of the editor and the overview doesn't match anymore.

**Comment by codebrainz** [comment]
I just made the Overview plugin because it was highly requested. There's many different Scintilla settings that would have to be duplicated to make it match 100%. As I don't personally use the plugin, I have no motivation to spend more time on it.

I would accept any pull requests that duplicate any/all of the Scintilla settings to the Overview though.

**Comment by Johndeep** [comment]
That's too bad, but somehow understandable that you don't use it, since it doesn't really help to navigate the code properly (doesn't matter if VSCode, KDevelop or other editors with similar Overview feature), since it just show you a miniature code on the right.

A nice feature on the other hand would be an "info scrollbar", which indicates the position of keywords and similar, just like Eclipse or Android Studio in example have it. But this for another issue thread maybe.
